controller/tests/array_action/ds8k/test_array_mediator_ds8k.py

```diff
@@ -70,6 +71,7 @@ def setUp(self):
         )
 
         self.array = DS8KArrayMediator("user", "password", self.endpoint)
+        self.volume_cache = self.array.volume_cache
```

Comment:
are you sure this uses the `patch.dict`?

Reply:
not relevant